.travis.yml

```diff
@@ -13,12 +13,11 @@ before_install:
   - sudo /etc/init.d/mysql stop
   - git clone https://github.com/FAForever/faf-stack.git faf-stack
       && pushd faf-stack
-      && git checkout v19.8.12
+      && git checkout v20.2.7
+      && export $(egrep -v '^#' .env.template | xargs)
```

Comment:
What does this line do? Set environment variables?

Reply:
Exactly. It seems to be necessary for `scripts/init-db.sh` now but apparently wasn't before.

Reply:
I see. Yea it seems I didn't have a `.env` file before, but now I was unable to rebuild the database until I copied it from `.env.template`.